File: tests/openmp_examples/synchronization/acquire_release_broke.4.c
#pragma omp begin declare variant match(device={kind(host)})
#pragma omp end declare variant
#pragma omp begin declare variant match(device={kind(nohost)})
#pragma omp end declare variant
#pragma omp parallel num_threads(2)
#pragma omp critical
#pragma omp atomic write
#pragma omp atomic read acquire
#pragma omp critical

File: tests/openmp_examples/synchronization/acquire_release_broke.4.f90
!$omp    parallel num_threads(2) private(thrd) private(tmp)
!$omp          critical
!$omp          end critical
!$omp          atomic write
!$omp          end atomic
!$omp            atomic read acquire
!$omp            end atomic
!$omp          critical
!$omp          end critical
!$omp    end parallel
